### ISSUE TITLE
Quicker access to "found" notes--open first instance

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#75]
```
Request to have first page opened after "Search all notes". Follow with
implied "Find" with same target string.
```

Original issue reported on code.google.com by `Phil.Fuh...@gmail.com` on 1 Jul 2009 at 5:15

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 8 Jul 2009 at 12:45
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 15 Jun 2010 at 6:13
- Added labels: **search**

**Comment by GoogleCodeExporter** [comment]
```
First note now opens automatically after search.  See keepnote 0.7.2:
http://keepnote.org/keepnote/download-test/
```

Original comment by `matt.rasmus@gmail.com` on 24 Apr 2011 at 8:43
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This is now implemented in 0.7.2
```

Original comment by `matt.rasmus@gmail.com` on 14 May 2011 at 7:03
- Changed state: **Done**
